### DISCUSSION TITLE
Add "docker image prune" query for moving Container Manager

### DISCUSSION THREAD

**Discussion by Dennis90BW** [007revad/Synology_app_mover#107] [Ideas]
Hello Dave,

nice work! very handy script. Maybe you could add a query to clean-up docker images before stopping the Container Manager and start a backup. My backup takes ages because of millions of unnecessary docker subvolume files in the @docker directory. The docker image prune command should do the job. (See: https://docs.docker.com/reference/cli/docker/image/prune/)

**Comment by 007revad** [comment]
Interesting.

My Synology_docker_cleanup script uses the following to delete dangling images:
```
docker rmi "$(docker images -f "dangling=true" -q)"
```
but it looks like I could have just used:
```
docker image prune
```
On my test Synology I had 5 images and 3 containers and ran `docker image prune --all`
```
# docker image prune --all
WARNING! This will remove all images without at least one container associated to them.
Are you sure you want to continue? [y/N] y
Deleted Images:
deleted:
deleted:
deleted:
deleted:
deleted:
deleted:

Total reclaimed space: 0B
```
It appears to have deleted 6 images, but didn't show which images. I assume 4 of those were dangling images. The 2 unused images totalled 64MB so I'm not sure why it says `Total reclaimed space: 0B`

I think I'll use `docker image prune` in the script to just delete dangling images.

**Comment by 007revad** [comment]
I've added "docker image prune" in v3.2.61

https://github.com/007revad/Synology_app_mover/releases/tag/v3.2.61
